tests/integration/cli/tests/run.rs

```diff
@@ -546,3 +546,46 @@ fn run_no_start_wasm_report_error() -> anyhow::Result<()> {
     assert_eq!(result.contains("Can not find any export functions."), true);
     Ok(())
 }
+
+// Test that wasmer can run a complex path
+#[test]
+fn test_wasmer_run_complex_url() -> anyhow::Result<()> {
+    let temp_dir = tempfile::tempdir()?;
+    let path = temp_dir.path().to_path_buf();
+    let root = std::env::home_dir().unwrap_or(path);
```

Comment:
We shouldn't rely in the homedir to test. We should rely on files within the repository itself

Reply:
ok done